democracylab/settings.py

```diff
@@ -99,6 +99,14 @@
 
 LOGIN_REDIRECT_URL = '/'
 
+#Caching number of tag counts only for now - change this if other things are db-cached
+CACHES = {
+    'default': {
+        'BACKEND': 'django.core.cache.backends.db.DatabaseCache',
+        'LOCATION': 'active_tag_counts',
```

Comment:
We may want to use this cache table for other things, so let's name it something more generic.